pkg/aws_config_server/types.go

```diff
@@ -36,6 +37,20 @@ func (a *AWSConfig) GetAccounts() []AWSAccount {
 	return accounts
 }
 
+func (a *AWSConfig) GetRoleNames() []string {
```

Comment:
Doesn't the AWSProfile struct now contain the role names? Should we use those instead?